### DISCUSSION TITLE
Ad an "additional product" to an order even if it not intendet

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretix/pretix#3993] [Ideas]
### Problem and impact

I have an "additional product A" that cannot actually be selected for product B during order process because i didn´t add it in the product settings.

But i want to manually add it to an existing order.

At the moment there is the error
"Dieses Zusatzprodukt kann nicht zu diesem Basisprodukt hinzugefügt werden."

my bugfix is now: manually add the additional product in the product settings, change the order the way I want it and delete it in the product settings again.

### Expected behaviour

It would be nice if pretix would notify me "This additional product should not be added to this basic product" and ask me "would you like to add it anyway?"
Similar to "Allow overbooking of contingents for this action"

### Version

2024.2.0

**Comment by raphaelm** [comment]
I get it, the idea was that adding a product that was not intended to ever be used as an add-on at all there could be quite unexpected consequences.
